docs/project/reward-mechanism-v2.mdx

```diff
@@ -5,74 +5,232 @@ sidebar_label: Reward Mechanism v2
-  - **Hardware Class**: Each type of approved hardware (weather station) will be assigned a specific hardware class. Hardware Classes will be used in the future to award more tokens to weather station hardware that has special capabilities, increased sensor accuracy or range, new sensor types and other capabilities that make them more valuable to the network.
-  - **Location**: WeatherXM places great importance on station continuity, necessitating that a station remains stationed at a single location for its entire duration. When a station undergoes relocation, its seniority is reset, aiming to discourage frequent movements by imposing a relocation fee upon the Network.
+This is a short summary of the criteria used:
 
-Each cell has a pre-defined *cell capacity* that depends on its weather characteristics. The cell capacity is the maximum number of stations that may receive rewards in a specific cell. If more than N=cell capacity stations are active in a cell during a specific day, the first N stations (ordered first by *reward_score*, then by seniority) will receive rewards. In over-crowded cells, while senior stations have a small advantage, it is important to point out that a newer station with better data will receive rewards before a senior station with lower quality data.
+* **Proof-of-Location (PoL)**: An algorithm that evaluates the data about the location of the weather station, its accuracy, consistency, and unison with the location the owner registered on the weather station claiming. It generates a score that denotes confidence about the weather station's location. WeatherXM places great importance on station continuity, necessitating that a station remains stationed at a single location. When a weather station undergoes relocation, its PoL score reduces to 0 and its seniority in the Cell is reset, aiming to discourage frequent movements and sustain a stable Network.
 
-:::note For example:
-If the capacity of a cell is 5, but has 6 stations with a random *reward_score*, then only the first 5 devices with the highest *reward_score* will be rewarded. If there are two stations in the 5th place, with exactly the same *station_reward_score*, then the more senior of the two will get the rewards
-:::
+  \
+* **Quality-of-Data (QoD)**: An algorithm that evaluates the quality of weather data provided by a weather station. It generates a score that denotes the confidence about the quality of the weather data for each weather station. WeatherXM values weather stations that produce meaningful and usable data and the QoD score is an attempt to quantify this metric. The end goal is to encourage weather station owners to do their best to comply with WeatherXM's guidelines in order to consistently achieve the best QoD score possible.
+
+  \

```

Comment:
What are these used for @tmavroeid ? I think these `\` on that list are unnecessary.